types/options.d.ts

```diff
@@ -177,7 +177,7 @@ export interface WatchOptions {
 }
 
 export interface WatchOptionsWithHandler<T> extends WatchOptions {
-  handler: WatchHandler<T>;
+  handler: WatchHandler<T> | string;
```

Comment:
The WatchHandler type itself should include string as it appears in other places

Reply:
Moved it there @posva